### ISSUE TITLE
Eeasier way to edit value in cookie array?

### ISSUE THREAD

**Issue by TechOverflow** [js-cookie/js-cookie#482]
I'm creating a cookie with an array like so

`var storeData = new Array();
            storeData[0] = provider;
            storeData[1] = providerName;
            storeData[2] = postnum;
            storeData[3] = description;
            storeData[4] = updates;
            storeData[5] = opened;
            
            Cookies.set(postnum, JSON.stringify(storeData));`

This works just fine, but to edit a value, say `storeData[5]`, I have to get the cookie, edit the value, then set the cookie again. Is there a way to shorten this, ie. Cookies.edit(postnum[0], newValue), before I write a special function to do this?

**Comment by carhartl** [comment]
No, there isn‘t. You might be able to leverage `Proxy` to solve this in an elegant way (observe change, whenever there is, write the stringified array to the cookie).

https://stackoverflow.com/questions/35610242/detecting-changes-in-a-javascript-array-using-the-proxy-object
